src/software/ai/passing/cost_function.cpp

```diff
@@ -100,22 +96,30 @@ double ratePassShootScore(const Field& field, const Team& enemy_team, const Pass
         net_percent_open = open_angle_to_goal.toDegrees() / goal_angle.toDegrees();
     }
 
-    // Create the shoot score by creating a sigmoid that goes to a large value as
-    // the section of net we're shooting on approaches 100% (ie. completely open)
-    double shot_openness_score = sigmoid(net_percent_open, 0.5, 0.95);
+    double shot_openness_score = sigmoid(net_percent_open, 0.2, 0.95);
 
     // Prefer angles where the robot does not have to turn much after receiving the
     // pass to take the shot (or equivalently the shot deflection angle)
-    Angle rotation_to_shot_target_after_pass = pass.receiverOrientation().minDiff(
-        (shot_target - pass.receiverPoint()).orientation());
+    //
+    // Receiver robots on the friendly side, almost always, need to rotate a full 180
+    // degrees to shoot on net. So we relax that requirement for both receiver and ball
+    // locations on the friendly side
+    if (pass.receiverPoint().x() < 0 || ball.position().x() < 0)
+    {
+        ideal_max_rotation_to_shoot_degrees = 180;
+    }
```

Comment:
Ah, seems this partly deals with the point I was talking about above. Did you notice any issues from this causing infinite changes in slope in the cost function? I'd suggest carefully looking at the cost function here a bit to ensure this isn't going to be a issue in the future (if you haven't already ofc).

Reply:
The slope is really high on that boundary (maybe even infinite), but this is more just a stop gap before doing a pass on all the cost functions and tuning them. Adam doesn't seem to care either way because this lines up with the zone boundaries

This will be fixed as part of https://github.com/UBC-Thunderbots/Software/issues/1987